.github/workflows/changeset.yaml

```diff
@@ -11,6 +11,7 @@ jobs:
     outputs:
       hasChangesets: ${{ steps.changesets.outputs.hasChangesets }}
       hasPublished: ${{ steps.changesets.outputs.published }}
+      publishedPackages: ${{ steps.changesets.outputs.publishedPackages }}
```

Comment:
Is this needed?

Reply:
Kinda want to try it out for the description of the deployment later